Author: Michael R. Crusoe <crusoe@debian.org>
Description: Don't build the wasm feature

The js feature of Debian's package of the getrandom crate is not yet available
--- liblzma.orig/Cargo.toml
+++ liblzma/Cargo.toml
@@ -33,7 +33,6 @@
     "lzma",
     "xz",
     "encoding",
-    "wasm",
 ]
 categories = [
     "compression",
@@ -99,11 +98,6 @@
     "tokio-io",
     "futures",
 ]
-wasm = ["liblzma-sys/wasm"]
 
 [target.'cfg(not(target_arch = "wasm32"))'.dev-dependencies.tokio-core]
 version = "0.1.17"
-
-[target.'cfg(target_arch = "wasm32")'.dev-dependencies.getrandom]
-version = "0.2"
-features = ["js"]
